addons/storyshots/src/html/renderTree.js

```diff
@@ -0,0 +1,16 @@
+import { document, Node } from 'global';
+
+function getRenderedTree(story, context) {
+  const component = story.render(context);
+
+  if (component instanceof Node) {
+    return component;
+  }
+
+  const section = document.createElement('section');
+  section.innerHTML = component;
+
+  return section.firstChild;
```

Comment:
One should be able to render more than one element. So I think we should use `section.childNodes` instead

Reply:
There is a problem of creating a tree for the array of nodes, so I will return a wrapper in case of a few child nodes

Reply:
can [`DocumentFragment`](https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment) help?

Reply:
```
const fragment = document.createDocumentFragment()
section.childNodes.forEach(node => fragment.appendChild(node))
```

Reply:
Nope, didn't help.